CHANGELOG.md

```diff
@@ -4,6 +4,15 @@
 
 * Dependency updates (Kafka 3.6.1, Kubernetes configuration provider 1.1.2, Vert.x 4.5.2, Netty 4.1.106.Final, Jackson FasterXML 2.16.1, Micrometer 1.12.2)
 * Fixed missing messaging semantic attributes to the Kafka consumer spans
+* Removed the dependency on OkHttp and thus Kotlin.
+* Dependency update (OpenTelemetry Semconv from 1.19.0-alpha to 1.21.0-alpha)
+  * The following span attributes are being replaced replacements:
+    1. `http.method` is being replaced with `http.request.method`
+    2. `http.url` is being replaced with `url.scheme`, `url.path` & `url.query`
+    3. `messaging.destination` is being replaced with `messaging.destination.name`
+    4. `http.status_code` is being replaced with `http.response.status_code`
+  * The following span attributes are slated for removal:
+    1. `messaging.destination.kind` & `messaging.destination.kind=topic`
```

Comment:
why it's mentioned twice? I mean the attribute name and then the same with the value which in our case will always be `topic`.

Reply:
I phrased it clumsily I was trying to capture that both the key and value types were being deprecated.